samples/ReactivityDemo/ReactivityDemo.csproj

```diff
@@ -1,14 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
 
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
     <ImplicitUsings>enable</ImplicitUsings>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="6.0.1" />
-    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="6.0.1" PrivateAssets="all" />
-    <PackageReference Include="MvvmBlazor" Version="2.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly" Version="8.0.8" />
+    <PackageReference Include="Microsoft.AspNetCore.Components.WebAssembly.DevServer" Version="8.0.8" PrivateAssets="all" />
+    <PackageReference Include="MvvmBlazor" Version="6.0.6" />
```

Comment:
Changing the version of MvvmBlazor, requires rewriting [the comparison document](https://github.com/phorks/phork-blazor-reactivity/blob/main/docs/COMPARISON.md), as it specifically compares the library to the 2.0.0 version of MvvmBlazor. As this project is only a demo, there is no need to upgrade it, could you rebase your branch to remove the commit that changes the demo project?